src/sentry/db/models/fields/bounded.py

```diff
@@ -110,3 +110,6 @@ class BoundedBigAutoField(BoundedAutoField):
 add_introspection_rules([], ["^sentry\.db\.models\.fields\.bounded\.BoundedBigIntegerField"])
 add_introspection_rules([], ["^sentry\.db\.models\.fields\.bounded\.BoundedPositiveIntegerField"])
 add_introspection_rules([], ["^sentry\.db\.models\.fields\.pickle\.UnicodePickledObjectField"])
```

Comment:
Were you going to remove this? They seem to have been added into the `fields/encrypted.py`.